packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -15,17 +15,30 @@ import ArbitrumNovaLogo from '../assets/ArbitrumNovaLogo.webp'
 
 const INFURA_KEY = process.env.REACT_APP_INFURA_KEY as string
 
+const MAINNET_INFURA_RPC_URL = `https://mainnet.infura.io/v3/${INFURA_KEY}`
+const GOERLI_INFURA_RPC_URL = `https://goerli.infura.io/v3/${INFURA_KEY}`
+
 if (!INFURA_KEY) {
```

Comment:
NIT: We should probably move this check above line 18. And we might want to cast `const INFURA_KEY = .... as string | null` instead?